components/ww3/cime_config/buildnml

```diff
@@ -50,8 +51,9 @@ def buildnml(case, caseroot, compname):
     # Verify wav grid is supported
     #--------------------------------------------------------------------
 
-    wav_grid_supported = ("null", "wQU225EC60to30sp32x36", "wQU225EC60to30", "wQU100", "wQU225EC30to60E2r2")
-    expect(wav_grid in wav_grid_supported, "WAV_GRID '{}' is not supported in ww3. Choose from: '{}'".format(wav_grid,' ,'.join(wav_grid_supported)))
+    wav_grid_supported = ("null", "wQU225EC60to30sp32x36", "wQU225EC60to30sp50x36", "wQU100sp50x36", "wQU225EC30to60E2r2sp50x36","wQU225EC30to60E2r2sp36x36","wQU225EC30to60E2r2sp25x36")
```

Comment:
maybe we should list these on separate lines:
```suggestion
    wav_grid_supported = ("null",
                                          "wQU225EC60to30sp32x36", 
                                           "wQU225EC60to30sp50x36", 
                                           "wQU100sp50x36",           
                                           "wQU225EC30to60E2r2sp50x36",
                                           "wQU225EC30to60E2r2sp36x36",
                                           "wQU225EC30to60E2r2sp25x36")
```
with all the same alignment (I can't seem to get it to format correctly)

Reply:
sure